docker-hub/builds/index.md

```diff
@@ -34,16 +34,16 @@ repositories with Automated Builds configured.
 
 If you have automated tests configured, these run after building but before
 pushing to the registry. You can use these tests to create a continuous
-integration workflow where a build that fails its tests does not push the built
-image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
+integration workflow where a build that fails its tests doesn't push the built
+image. Automated tests don't push images to the registry on their own. [Learn about automated image testing](automated-testing.md).
 
-Depending on your [plan](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"},
+Depending on your [subscription](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"},
 you may get concurrent builds, which means that `N` autobuilds can be run at the
-same time. `N` is configured by the plan that you subscribe to. Once `N+1` builds
-are running, any additional builds are queued to be run later.
+same time. `N` is configured according to your subscription. Once `N+1` builds
+are running, any additional builds goes into a queue to be run later.
 
-The total number of pending builds in the queue is limited to 30 and further
-requests will be discarded. The number of concurrent builds for Pro is 5 and
+The maximum number of pending builds in the queue is 30 and discards further
```

Comment:
```suggestion
The maximum number of pending builds in the queue is 30 and Docker Hub discards further
```
I think some subject should be added. I'm not 100% sure it's Docker Hub though.

Reply:
Good point. I think it is Hub, I'll confirm with Marc.

Reply:
Will update in new commit.